packages/shared/src/components/search/SearchProgressBar.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+
+export interface SearchProgressBarProps {
+  progress: number;
+}
+
+export const SearchProgressBar = ({
+  progress,
+}: SearchProgressBarProps): ReactElement => {
+  return (
+    <div className="flex relative gap-6 h-2" data-testId="SearchProgressBar">
+      <div className="relative flex-1 h-full bg-white rounded-full">
```

Comment:
The background is actually different.
(Also the animation, but let's leave that out for another ticket)
<img width="671" alt="Screenshot 2023-08-02 at 12 11 50" src="https://github.com/dailydotdev/apps/assets/554874/bb1cf4ff-27bd-4c85-a58a-303f20a83e1c">

Reply:
@TomDowling I still see bg-white here, not sure it's fixed?